### ISSUE TITLE
#852 not resolved with build 4.5.8

### ISSUE THREAD

**Issue by rrrevin** [TV-Rename/tvrename#901]
Just tried the new 4.5.8 build, and the problem described in #852 is not resolved.
When selecting "Scan" -> "Recent Scan".. the system happily does it progress thing and the list of possible choices is shown.

![image](https://user-images.githubusercontent.com/49560189/153475591-1a4daf62-9d01-441e-a9fe-9940abec5756.png)

"Check" -> unselecting "Show all", should unselect all items. But it's still broken. Some items are refreshed correctly and show as unselected (in red below)

![image](https://user-images.githubusercontent.com/49560189/153476048-607073c7-73dd-45c3-994d-a195532e4715.png)

But all the other lines still show as selected. I have to mouse over each line to get the tick boxes to update, as shown below. In the background the items doesn't seems to be selected, but it is displayed as if it is.

![image](https://user-images.githubusercontent.com/49560189/153476555-0d289a58-30aa-4849-883a-30f8baed386e.png)

**Comment by SirSparkles** [comment]
Hmm - the behaviour you see does not match what I get. When I 'Check All' I have a small delay ~1 sec and then they all enable / disable as one.

1 second is for ~2000 actions. Much quicker if the number of action is smaller.

Anyone else seeing similar problems?

**Comment by rrrevin** [comment]
[crcarley](https://github.com/crcarley) said they did in #852.

**Comment by SirSparkles** [comment]
Yeah - am interested to know whether the changes I've put in place improved it for them

**Comment by rrrevin** [comment]
Do you have a dependency list? I'm wondering if a cause might be some differing version of an underlying library someplace?

**Comment by rrrevin** [comment]
Maybe in the log also display version info?
```
2022-02-28 12:56:35|INFO| Loaded file with updates until 1969-12-31 5:00:00 PM 
2022-02-28 12:56:35|WARN| Updated path for Assembly: C:\Program Files (x86)\TVRename\x64\CefSharp.Core.Runtime.dll 
2022-02-28 12:56:35|INFO| Updated path for BrowserSubprocess: C:\Program Files (x86)\TVRename\x64\CefSharp.BrowserSubprocess.exe 
2022-02-28 12:56:35|INFO| Updated path for LocalesDirPath: C:\Program Files (x86)\TVRename\x64\locales 
2022-02-28 12:56:35|INFO| Updated path for ResourcesDirPath: C:\Program Files (x86)\TVRename\x64 
2022-02-28 12:56:36|INFO| Dependencies all found 
```

**Comment by rrrevin** [comment]
Extra info.. I installed older version to try and figure exactly when this started.
4.5.3 -> works as expected. No issues.
4.5.4 -> After Recent scan, app crashes. Can't test.
4.5.5 -> No longer works. This is the version that I can state the error starts.

What ever changed between 4.5.3 and 4.5.5 is causing the issue.
Hope this helps.

**Comment by rrrevin** [comment]
Still happens with 4.6.1

**Comment by SirSparkles** [comment]
@rrrevin @crcarley
You still seeing these issues with 4.6.6? They are difficult for me to fix as I can't reproduce them

**Comment by rrrevin** [comment]
This still occurs in 4.7

**Comment by rrrevin** [comment]
This seems resolved with 4.8.4. That's the only version that I've tested since my comment above. I'll use over the next few weeks and follow up.
